pkg/api/defaults.go

```diff
@@ -67,6 +67,8 @@ var DefaultOperatorFlags OperatorFlags = OperatorFlags{
 
 	"aro.machineset.enabled": FLAG_TRUE,
 
+	"aro.machinehealthcheck.enabled": FLAG_FALSE,
```

Comment:
Why false by default?

Reply:
For an initial roll out we would manually enable this on a small set of clusters, monitor how it is working and eventually flip this to default as true.